### PR TITLE
CVE-2011-4137

### PR DESCRIPTION
**Pull request by sk6786**
CVE edit for verify_exists timeout vulnerability

### CODE REVIEW

File: cves/CVE-2011-4137.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: A method called verify_exits uses a python library that calls the arbitrary URL without a timeout. This can be a security threat as someone can cause denial of service by simply passing a url into the verify_exists which is associated with a slow response or a large amount of data sent.
```
Comment (bxj3964) [inline cves/CVE-2011-4137.yml]: You probably want a manual newline here as well.

File: cves/CVE-2011-4137.yml [hunk 2]
```diff
@@ -181,20 +171,21 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: it is called the models system. I know ths as I referred to the source code as well as on their webblog which announced their vulnerability https://www.djangoproject.com/weblog
```
Comment (bxj3964) [inline cves/CVE-2011-4137.yml]: Typo with this in the second sentence.

File: cves/CVE-2011-4137.yml [hunk 3]
```diff
@@ -244,8 +235,8 @@ lessons:
     applies:
     note:
   distrust_input:
-    applies:
-    note:
+    applies: true
+    note: this one definetely applies. Don't trust that the user is going to supply a url that is safe. Have a check that sees how system handles malicous url. Classic case of not modeling abuse cases.
```
Comment (bxj3964) [inline cves/CVE-2011-4137.yml]: Typo in 'definitely'

File: cves/CVE-2011-4137.yml [hunk 4]
```diff
@@ -204,8 +195,8 @@ i18n:
 
     Answer should be boolean. Write a note about how you came to the conclusions
     you did.
-  answer:
-  note:
+  answer: false
+  note: no this error was not associated with internationalization, as the method would not fail due to some faults in internationalization, but instead the misshandling of timeouts in the library
```
Comment (bxj3964) [inline cves/CVE-2011-4137.yml]: Small typo in 'mishandling' at the end.

File: cves/CVE-2011-4137.yml [hunk 5]
```diff
@@ -148,8 +138,8 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: I believe if the test case had an input url that directed a slow loading page, and the test case had a timeout, then the developers could see the test failed and understand that there is a vulnerability in the verify_exists method.
```
Comment (jsj1027) [inline cves/CVE-2011-4137.yml:145]: Maybe understood rather than understand?

File: cves/CVE-2011-4137.yml [hunk 6]
```diff
@@ -161,8 +151,8 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: I have checked commits and bug reports. It only states that the vulnerability leads to a denial of service attack. No mention of specification.
```
Comment (jsj1027) [inline cves/CVE-2011-4137.yml]: Maybe you can list which bug reports you looked at?

File: cves/CVE-2011-4137.yml [hunk 7]
```diff
@@ -289,8 +280,8 @@ CWE_instructions: |
 
   Just the number here is fine. No need for name or CWE prefix. If more than one
   apply here, then choose the best one and mention the others in CWE_note.
-CWE:
-CWE_note:
+CWE: 400
+CWE_note: Uncontrolled resource consumption. This is applicable because the url library didn't have a timeout so the url could potentially send large amounts of data causing resource to be consumed.
```
Comment (jsj1027) [inline cves/CVE-2011-4137.yml:288]: I give this 3 upvotes

File: cves/CVE-2011-4137.yml [hunk 8]
```diff
@@ -15,24 +15,24 @@ curated_instructions: |
   integrity checks on this file to make sure you fill everything out properly.
```
Comment (bxj3964) [inline cves/CVE-2011-4137.yml:15]: I upvote this 2.

**Pull request merged**